spring-boot-project/spring-boot-autoconfigure/src/main/java/org/springframework/boot/autoconfigure/elasticsearch/ElasticsearchRestClientProperties.java

```diff
@@ -56,6 +56,14 @@
 	 * Read timeout.
 	 */
 	private Duration readTimeout = Duration.ofSeconds(30);
+	/**
+	 * Sniffer interval
```

Comment:
Configuration property description ends with a dot. Please consider reviewing the description to better explain what that does.